slidedeck/src/slides/01-layout-tools/03-testing.pug

```diff
@@ -32,6 +32,19 @@ section
     li axios is a promise based HTTP client for the browser and node.js
     li moxios is for mocking axios requests while testing
 
+section
+  p Making a request
+  pre.
+    <code class="javascript" data-trim>
+      axios.get('/api/users')
+        .then(function(response) {
```

Comment:
I don't think you wrote these, but I think we're safe to use arrow functions now in all of our examples.

So... Todo moreso than "fix this now" ha